docs/source/develop/api/queries.rst

```diff
@@ -58,11 +58,23 @@ After ordering is specified, pagination can be executed:
 .. code-block:: proto
 
     message TxPaginationMeta {
-        uint32 page_size = 1;
-        oneof opt_first_tx_hash {
-            string first_tx_hash = 2;
-        }
-        Ordering ordering = 3;
+      uint32 page_size = 1;
```

Comment:
Please DRY. Let this code-block be sourced from original queries.proto. Let future changes to queries.proto be synchronized automatically.
Looks like you'll need to move `TxPaginationMeta`, please consider ways to do.

Reply:
I think that other code fields in this .rst file could be done this way. Should I modify them also?

Reply:
Sorry for breaking in: Changing rest of fields in the .rst file would be not hard, but it would make PR harder to verify and bigger. So I suggest not to change other fields in .rst file in the PR.

The pointed issue is corrected for TxPaginationMeta so I'm marking as resolved.